package.json

```diff
@@ -23,8 +28,28 @@
   "homepage": "https://github.com/everydayhero/react-radial-list#readme",
   "devDependencies": {
     "ava": "^0.16.0",
+    "babel-polyfill": "^6.16.0",
     "boiler-room-builder": "^1.0.0-18",
     "cxs": "^1.0.1",
-    "enzyme": "^2.4.1"
+    "enzyme": "^2.4.1",
+    "escape-string-regexp": "^1.0.5",
+    "jsdom": "^9.6.0"
+  },
+  "babel": {
+    "presets": [
+      "es2015",
+      "stage-0",
```

Comment:
Should we start using preset latest?
https://babeljs.io/docs/plugins/preset-latest/